cloudinit/net/dhcp.py

```diff
@@ -222,7 +222,7 @@ def dhcp_discovery(dhclient_cmd_path, interface, cleandir):
     util.subp(['ip', 'link', 'set', 'dev', interface, 'up'], capture=True)
     cmd = [sandbox_dhclient_cmd, '-1', '-v', '-lf', lease_file,
            '-pf', pid_file, interface, '-sf', '/bin/true']
-    util.subp(cmd, capture=True)
+    pout = util.subp(cmd, capture=True)
```

Comment:
`util.subp` will always return (stdout, stderr), so we can make this:

```suggestion
    _out, err = util.subp(cmd, capture=True)
```